pandas/_libs/window.pyx

```diff
@@ -650,20 +651,20 @@ cdef inline double calc_var(int64_t minp, int ddof, double nobs,
     return result
 
 
-cdef inline void add_var(double val, double *nobs, double *mean_x,
+cdef inline void add_var(const double val, double *nobs, double *mean_x,
```

Comment:
is this const needed?

0.23.4 is still on an older cython

Reply:
No, just me trying random stuff to fool the compiler, I'll back that out